components/governance/Timeline.vue

```diff
@@ -97,4 +97,15 @@ export default {
   font-size: 12px;
   color: var(--dim);
 }
+
+@media (max-width: 768px) {
+  .timeline {
+    max-width: 1024px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    margin: 0 auto;
+    width: 100%;
+  }
+}
```

Comment:
I think the timeline display is more of a design problem, common solution is changing orientation

Reply:
![E584BFAC-B048-4694-82B2-02F0D1945640](https://user-images.githubusercontent.com/17264716/114157605-63785600-9956-11eb-9c07-8e7ab0895cd8.jpeg)